### PR TITLE
lean app blog series

### PR DESCRIPTION
**Pull request by vikrant-vendia**

### CODE REVIEW

File: posts/2022-02-03-lean-app.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+The best application developers are leading with judiciousness, maniacally focusing on solving the toughest problems in the leanest, most elegant way possible. As developers, we place a premium on the idea that less is more. We are precision coding. The code we write is entirely aimed at delivering unique value – leaving undifferentiated and previously-solved problems to the rich set of cloud services at our fingertips. 
+
+We are building powerful, lean applications.  
+
+
+### A four-part blog series
+
+This blog series covers the lean app movement in four parts.  
+
+**[Part 1 – Lessons from the Past.](Vendia.net/blog/lean-app-part-1)** This section takes a look at the trendlines that have made it possible to develop lean apps. I highlight the technology shifts that make it possible for us to apply an extreme focus on building software based on customer value, rather than technological ownership
```
Comment (DavidWells) [inline posts/2022-02-03-lean-app.md:24]: `Vendia.net/blog/lean-app-part-1` and other links like it need to be full URLs e.g. `https://www.vendia.net/blog/lean-app-part-1`

**Comment by caroll-vendia** [comment]
I'll go fix the link so they go to full URLs and will remove links to blogs 2,3,4 until they are live.

File: posts/2022-02-03-lean-app.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+**[Part 2 – The Elements of a Lean App](Vendia.net/blog/lean-app-part-2)**. In Part 2 I define the ideal model of what a software application should look like. For an industry that has for so long defined “software” as “stuff that runs on a machine”, it can be hard to wrap your head around the idea of giving up on that paradigm. This post clearly outlines the elements of a lean app, including data model, integrity constraints, state transitions, and connectors.
+
+**[Part 3 – Letting Infrastructure do the Dirty Work](Vendia.net/blog/lean-app-part-3)**. A fundamental tenet of the Lean App Movement is to stop the undifferentiated heavy lifting and leave that to the cloud infrastructure providers. But for that to be possible, the infrastructure on which the application depends needs to start doing a whole lot more than it does today. This post covers what infrastructure needs to do automatically and without impacting developers or operators.
```
Comment (caroll-vendia) [inline posts/2022-02-03-lean-app.md:28]: remove link and add "(coming soon!)"

File: posts/2022-02-03-lean-app.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+**[Part 4 – The Lean App Manifesto](Vendia.net/blog/lean-app-part-4)**. What does a lean world look like? In some ways, it’s a continuation of everything that’s come before: open source software, the public cloud, Serverless. Each of these, in some way, helped to “lean out” some aspect of application development, without getting quite all the way there. The lean app movement is driven by a simple, overarching idea: Less over more. If a technology, process, approach, or vendor can do something effectively that’s not your core business, then let them. This post covers the six simple rules lean apps follow.
```
Comment (caroll-vendia) [inline posts/2022-02-03-lean-app.md:30]: remove link and add "(coming soon!)"

File: posts/2022-02-03-lean-app.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+
+### Get the whitepaper
+
+If you don’t want to flip through four separate posts, you can download the full white paper [here](vendia.net/lean-app).
```
Comment (caroll-vendia) [inline posts/2022-02-03-lean-app.md:40]: replace url:  https://www.vendia.net/resources/lean-apps

**Review by caroll-vendia — CHANGES_REQUESTED** [review]

File: posts/2022-02-03-lean-app.md (hunk 4 above)
Comment (vikrant-vendia) [reply posts/2022-02-03-lean-app.md:40]: updated

File: posts/2022-02-03-lean-app-part-1.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+### #1 Removal
+
+The most powerful “reduction” of all is when an entire area is permanently removed from a developer’s purview. The shift from on-prem data centers to the public cloud is a great example of removal: Wholesale problems (“Which router should we use?” “What’s our network backplane bandwidth between data centers?” “How much rack space do we have left on the west coast?”) that used to consume precious IT time, people, and money have vanished…or at least been condensed into paying a public cloud provider’s margins, which are usually covered by the shift from buying peak capacity to paying for actual use.
+
+
+### #2 Reduction through standardization 
+
+Where things can’t be taken away entirely from the application stack, the next best thing is to focus a company’s (and developer’s) resources on the elements that differentiate their customer outcomes, rather than the undifferentiated “commodity” pieces. 
+
+ \
```
[the remark this answers is not in the document]
Comment (vikrant-vendia) [reply posts/2022-02-03-lean-app-part-1.md]: Where is ""?

**Pull request merged**